tests/sles4sap/monitoring_services.pm

```diff
@@ -88,7 +89,8 @@ sub configure_hanadb_exporter {
               . ' op monitor interval=10'
               . ' meta target-role=Stopped');
 
-        assert_script_run "crm configure colocation col_exporter_$args{rsc_id} +inf: $hanadb_exp_rsc:Started $hanadb_msl:Master";
+        my $role = is_sle(">=15-SP4") ? "Promoted" : "Master";
```

Comment:
maybe we can move this line near to `our $resource_aliase = is_sle(">=15-SP4") ? 'cln' : 'msl';` . In this way we keep in a single place each special setting activated by `is_sle(">=15-SP4")`

Reply:
Done. Thanks